geometry_msgs/Pose pose_hand
geometry_msgs/Pose pose_desired
bool ObjOrObst
uint16 WhichArm
